README.md

```diff
@@ -16,6 +14,14 @@ The current version is built for Mac OS 10.12 SDK (as of version 2.4.7). A compi
 2. Under `Assets`, select `atemOSC.dmg`
 3. Double-click the downloaded DMG, drag the application to your Applications folder, then launch it from the Launchpad.
 
+## Setup and Usage
+
+AtemOSC is a proxy, listening for commands following the [OSC protocol](http://opensoundcontrol.org/introduction-osc) and executing those commands on Blackmagic video switchers.  You just have to tell atemOSC where the switcher is and what port to listen on, and then send commands to the IP address of the computer running atemOSC on the specified port.  If you set an outgoing IP address and port, atemOSC will send status updates and feedback OSC messages to the device and port you specified.
```

Comment:
"where the switcher is and what `local` port to listen on"

"running atemOSC on the `port you specified`."

"updates and feedback OSC messages to the `IP address` and port you specified"